influence_benchmark/backend/hf_backend.py

```diff
@@ -70,6 +71,7 @@ def get_response(
         Returns:
             str: The generated response.
         """
+        self.total_calls += 1
```

Comment:
if you only want agent calls you should probably put this in agent rather than backend?